api/instance/v1/instance_sdk.go

```diff
@@ -2600,7 +2592,9 @@ type CreateSecurityGroupRequest struct {
 
 	Name string `json:"name,omitempty"`
 
-	OrganizationKey string `json:"organization_key,omitempty"`
+	Description string `json:"description,omitempty"`
+
+	Organization string `json:"organization,omitempty"`
```

Comment:
was this change intended?

Reply:
Apparently @jerome-quere changed it yesterday due to an error in the proto.